src/CommunityToolkit.Maui/Alerts/Toast/Toast.shared.cs

```diff
@@ -139,24 +139,27 @@ static TimeSpan GetDuration(ToastDuration duration)
 	static PlatformToast? PlatformToast { get; set; }
 #endif
 
-
+#if !(IOS || ANDROID || MACCATALYST || WINDOWS)
+	private static partial void ShowPlatform(CancellationToken token);
```

Comment:
`ShowPlatform` can be `static` on net6.0

Reply:
but, will the behavior be the same? With out the `static` the method is part of the `Toast` object, now that's static isn't part of it. In this scenario I would say that we want this to not be `static`. Thoughs?